wifi-subsys/components/network/src/sntp.c

```diff
@@ -0,0 +1,100 @@
+/* LwIP SNTP example
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
```

Comment:
remove this

Reply:
Well, if this job is based on that job the original header license should stay, only  add our copyright